packages/phrases/src/locales/de-DE/index.ts

```diff
@@ -0,0 +1,9 @@
+import errors from './errors';
+import translation from './translation';
+
+const en = Object.freeze({
+  translation,
+  errors,
+});
+
+export default en;
```

Comment:
This should've actually been:
```suggestion
const deDe: LocalePhrase = Object.freeze({
  translation,
  errors,
});

export default deDe;
```

But don't worry I can revise the code on behalf